api/tests/unit/organisations/invites/test_unit_invites_models.py

```diff
@@ -12,6 +12,7 @@
     from pytest_django.fixtures import SettingsWrapper
 
 
+@pytest.mark.django_db
```

Comment:
Odd, I'm not sure how this test was ever passing?

Reply:
It is caused by upgrading the lifecycle hook - they have "refactored" some bit

Reply:
Ah, ok! 👍